Docs/Deploy_Orleans_App_in_YAMS.md

```diff
@@ -10,80 +10,88 @@ Follow the *Hello World* tutorial described in Orleans [My First Orleans Applica
 
 Once you have the app ready, follow the steps below to deploy the app in YAMS:
```

Comment:
It would be great if you can mention somewhere in this tutorial (maybe at the beginning) that the Yams Orleans cluster should be deployed using the multiple roles sample (and if you can explain why that would be even better). Otherwise, users might deploy a single cluster as described in the *Deploy Yams* tutorial and will have difficulties deploying Orleans apps to Yams.